src/actix/api/service_api.rs

```diff
@@ -87,10 +91,42 @@ async fn get_locks(toc: web::Data<TableOfContent>) -> impl Responder {
     process_response(Ok(result), timing)
 }
 
+#[get("/log_level")]
+async fn get_log_level(handle: web::Data<LogLevelReloadHandle>) -> impl Responder {
+    handle
+        .get()
+        .map_or_else(plain_text_error, plain_text_response)
+}
+
+#[post("/log_level")]
+async fn set_log_level(
+    handle: web::Data<LogLevelReloadHandle>,
+    body: web::Bytes,
+) -> impl Responder {
+    str::from_utf8(&body)
+        .map_err(Into::into)
+        .and_then(|filter| handle.set(filter))
+        .map_or_else(plain_text_error, |_| HttpResponse::Ok().finish())
+}
```

Comment:
I think we should implement a JSON body and response here. That would make it consistent with the rest of the endpoints.

All other endpoints use JSON, except for `/metrics` or file downloads as that isn't possible.